bin/vaultclient

```diff
@@ -126,11 +126,15 @@ function handleVaultResponse(err, data) {
     });
 }
 
+function parseBigInt(stringValue) {
```

Comment:
what is the benefit of this function, it is just an alias for `BigInt` ?

Reply:
Updated